lib/widgets/sign_in_widget.dart

```diff
@@ -9,69 +11,94 @@ class _SignInState extends State<SignIn> {
-                decoration: InputDecoration(
-                  hintText: "Email",
-                ),
-              ),
-              TextField(
-                style: Theme.of(context).textTheme.bodyText1,
-                decoration: InputDecoration(
-                  hintText: "Password",
-                ),
-              ),
-              Divider(height: 18),
-              Container(
-                alignment: Alignment.centerRight,
-                child: Text(
-                  "Forgot Password?",
+      ),
+      body: SingleChildScrollView(
+        child: Container(
+          height: MediaQuery.of(context).size.height - 60,
+          alignment: Alignment.bottomCenter,
+          child: Container(
+            padding: EdgeInsets.symmetric(horizontal: 25),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                TextField(
                   style: Theme.of(context).textTheme.bodyText1,
+                  decoration: InputDecoration(
+                    hintText: "Email",
+                  ),
                 ),
-              ),
-              SizedBox(height: 18),
-              Container(
-                alignment: Alignment.center,
-                width: MediaQuery.of(context).size.width,
-                padding: EdgeInsets.symmetric(vertical: 20),
-                decoration: BoxDecoration(
-                    gradient: LinearGradient(colors: [
-                      const Color(0xff007EF4),
-                      const Color(0xff2A75BC)
-                    ]),
-                    borderRadius: BorderRadius.circular(30)),
-                child: Text(
-                  "Sign In",
+                TextField(
                   style: Theme.of(context).textTheme.bodyText1,
+                  decoration: InputDecoration(
+                    hintText: "Password",
+                  ),
                 ),
-              ),
-              SizedBox(height: 15),
-              Container(
-                alignment: Alignment.center,
-                width: MediaQuery.of(context).size.width,
-                padding: EdgeInsets.symmetric(vertical: 20),
-                decoration: BoxDecoration(
-                  color: Colors.white,
-                  borderRadius: BorderRadius.circular(30),
+                Divider(height: 22),
+                Container(
+                  alignment: Alignment.centerRight,
+                  child: Text(
+                    "Forgot Password?",
+                    style: Theme.of(context).textTheme.bodyText1,
+                  ),
                 ),
-                child: Text(
-                  "Sign In with Google",
-                  style: TextStyle(color: Colors.black87, fontSize: 17),
+                SizedBox(height: 22),
+                Container(
+                  alignment: Alignment.center,
+                  width: MediaQuery.of(context).size.width,
```

Comment:
Try to avoid hard coding of widget's size as possible. Find a flexible solution (minor)